api/PerformanceNavigationTiming.json

```diff
@@ -34,10 +34,10 @@
             "version_added": "43"
           },
           "safari": {
-            "version_added": false
+            "version_added": "11"
```

Comment:
This looks wrong in the other direction. I can't find any evidence that `PerformanceNavigationTiming` is supported in Safari. The subfeatures were added in #4039, but the linked release notes don't mention the Navigation Timing spec and [searching the WebKit source turns up no evidence it was ever implemented](https://github.com/WebKit/webkit/search?q=PerformanceNavigationTiming&unscoped_q=PerformanceNavigationTiming). I suspect the right thing here is to mark all of these features as `false` for the Safaris.

Reply:
Changes reversed!